certified-connectors/TractionGuest/apiDefinition.swagger.json

```diff
@@ -587,5 +676,23 @@
     }
   },
   "swagger": "2.0",
-  "tags": []
-}
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Publisher",
```

Comment:
@jzhang729 Can you please let us know what documentation you are referring to add metadata?
The `publisher` and `stackOwner` properties should be added to `apiProperties.json` file. Please remove it from Swagger.

Reply:
Hi @sriyen-msft . I think I was looking at other connectors like Envoy and Azure AD as examples.

Based on your feedback, I just pushed a new commit with those changes.